tests/test_cli.py

```diff
@@ -150,6 +153,28 @@ class Foo(SQLModel, table=True):
     )
 
 
+@pytest.mark.parametrize(
+    "cli_args, expected",
+    [
+        ([], None),
+        (["--thickmode"], True),
+        (["--thickmode", "true"], True),
+        (["--thickmode", "false"], False),
+        (
+            ["--thickmode", '{"lib_dir": "/foo", "driver_name": "v1"}'],
+            {"lib_dir": "/foo", "driver_name": "v1"},
+        ),
+    ],
+)
+def test_cli_thickmode(
+    cli_args: list[str],
+    expected: None | bool | dict[Any, Any],
+) -> None:
+    parser = create_parser()
```

Comment:
Please also implement the cli test like other cli tests are implemented (using a subprocess)